style.css

```diff
@@ -0,0 +1,168 @@
+html {
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+ul li {
+  list-style: none;
+  box-sizing: unset;
+  text-decoration: none;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.hidden {
+  display: none;
+}
+
+button {
+  border-style: none;
+}
+
+.menu {
+  display: flex;
+  flex-basis: 50%;
+  padding: 5px;
+}
+
+.welcome-logo {
+  display: flex;
+  flex: 1;
+  padding: 24px 36px 24px 24px;
+  margin: 0;
+}
+
+.welcome-logo a {
+  width: 90px;
+  height: 28px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.menu-button {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+}
+
+.menu-button ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 10px;
+  margin: 0;
+}
+
+.menu-button li {
+  align-self: center;
+  padding: 1px;
+}
+
+.bar {
+  background-color: rgb(32, 14, 14);
+  height: 0;
+  width: 16px;
+  left: 4px;
+  top: 17px;
+  border-radius: 5px;
+  border: 2px solid #fff;
+}
+
+/* ------- hero - section  ------ */
+
+.hero {
+  display: flex;
+  background-size: cover;
+  background-repeat: no-repeat;
+  padding: 58px 24px;
+  min-height: 85vh;
+  justify-content: center;
+  background-image: url(./images/one.png);
```

Comment:
- [ ] Your design is looking great, but this is the desktop version background, please use the mobile version one.